salt/metalk8s/addons/prometheus-operator/config/prometheus.yaml

```diff
@@ -7,6 +7,9 @@ spec:
   # Configure the Prometheus Deployment
   deployment:
     replicas: 1
+  config:
+    retention_time: 10d
+    retention_size: 0
```

Comment:
Since this file is `literal-include`d in our docs, would make sense to add a comment here saying that `0` means size-based retention is disabled.